modules/n1ql/pages/n1ql-language-reference/createindex.adoc

```diff
@@ -149,9 +149,9 @@ collection::
 If omitted, the default collection in the bucket's default scope is used.
 
 ====
-For example, `default:{backtick}travel-sample{backtick}` indicates the default collection in the default scope in the `travel-sample` bucket in the `default` namespace.
+For example, `default:{backtick}my-db{backtick}` indicates the default collection in the default scope in the `my-db` bucket in the `default` namespace.
 
-Similarly, `default:{backtick}travel-sample{backtick}.inventory.airline` indicates the `airline` collection in the `inventory` scope in the `travel-sample` bucket in the `default` namespace.
+Similarly, `default:{backtick}my-db{backtick}.samples.airline` indicates the `airline` collection in the `samples` scope in the `my-db` bucket in the `default` namespace.
```

Comment:
If the `my-db` is a bucket, do we really want to even use `my-db`? Should we even be mentioning a bucket if this is for Elixir?